### ISSUE TITLE
change from v5.2.1 to v6.0b2 causes "ValueError: invalid literal for int() with base 10"

### ISSUE THREAD

**Issue by DanCodigaMWRA** [oracle/python-cx_Oracle#86]
I have various python code modules written using calls to cx_Oracle v5.2.1, to run queries in an Oracle DB, and they work fine. 

I recently updated cx_Oracle to v6.02b. Some of the modules still work fine.

In the case of one of them, though, I am getting the error 
     ValueError: invalid literal for int() with base 10: '19.51'   
The error occurs after successfully making the connection and running the query, when I am calling Cursor.fetchall to retrieve the query results. 

The offending value (19.51) is the first entry in a column of the query output. It is a float and it seems like the error arises from an attempt to convert it to int (it may be getting treated as if it is a string?). 

This did not happen with v5.2.1.

--

I noticed Issue #34 seems closely related to this.

--

I am using: 

* Anaconda Python 3.5.2 -  64-bit

* cx_Oracle 6.0b2 
Installed using "conda update cx_oracle", when v5.2.1 was already installed.
(... and v5.2.1 was installed using "conda install -c dhirshfeld cx_oracle=5.2.1)

[And again, the problem/error does not occur for v5.2.1.]

* InstantClient 11.2.0.4.0
(installed by copying files from Oracle site to a local directory then adding that directory to my environment path)

* Windows 7 (64bit)

--

Thanks in advance for any suggestions to sort this out.

**Comment by anthony-tuininga** [comment]
Are you using cx_Oracle 6.0b2? Can you upgrade to 6.0.2 using the following command?

`python -m pip install cx_Oracle --upgrade`

and see if the problem persists? If it does, can you provide a sample that demonstrates the issue? Thanks.

**Comment by DanCodigaMWRA** [comment]
That seems to have solved the problem. 
Thanks very much. 

(Pardon my edits earlier, I had some confusion over v5.1.2 and v5.2.1. It turned out I was using v5.2.1, though i had forgotten that I obtained it from the dhirshfeld channel; but meanwhile conda itself was only able to find v5.1.2. So it took me a few check-backs to get it straight. In any case all that looks immaterial now, if 6.02 continues to perform for me I'll happily just stick with that.)

**Comment by anthony-tuininga** [comment]
Glad to hear that solved your issue. If you're satisfied, please close the issue. Thanks!

**Comment by DanCodigaMWRA** [comment]
Yes this solved my issue. Closing now.

( I will say though, because I am less familiar with pip I was hoping to avoid using it, i.e., to be able just stick with conda only. But "conda search cx_oracle" doesn't find 6.02 for me-- it only finds 6.0b2. Am curious why that is, but will save for later exploring that topic to come up to speed on it. Your instructions for pip were clear and they worked-- actually I just did "pip install cx_Oracle --upgrade" from an Anaconda prompt after activating the virtual environment I use-- so I'm all set for now. Maybe in the future it will be possible using conda. )
